checkov/sast/checks/rules/python/SuperuserPort.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+  - id: CKV_SAST_1
+    patterns:
+      - pattern: set_port($ARG)
+      - metavariable-comparison:
+          metavariable: $ARG
+          comparison: $ARG < 1024
+    message: module setting superuser port
+    languages:
+      - python
```

Comment:
Given the fact that this check is under `..rules/python/..`, if we want to implement this for TS would we need to duplicate this to `..rules/ts/` and just change the language?

Reply:
I think its going to be rare to have the exactly rule for multiple languages.
IMO in that case its ok to duplicate